python/ray/data/datasource/file_based_datasource.py

```diff
@@ -48,7 +48,7 @@
     import pyarrow
 
 
-logger = logging.getLogger(__name__)
+logger = DatasetLogger(__name__)
```

Comment:
Should we always be using `DatasetLogger` over `logging.getLogger`?

Reply:
i think for classes/logs related to Dataset operations (read, write, transforms, etc), we should use DatasetLogger.